### DISCUSSION TITLE
Self Hosting

### DISCUSSION THREAD

**Discussion by felixhj** [stefansundin/rssbox#69] [Q&A]
Hi Stefan,

I'm really interested in this project - for me it is about being able to control the content I consume on the internet. I have no coding experience but I would like to self-host. I have given it a good shot today with what I can work out for myself but am pretty out of my depth. I just wondered if you might be able to give a (very) brief overview of the process so I can research this further myself?

So far today I have signed up for heroku, downloaded the CLI and dependencies and the latest heroku version of rssbox. I am trying to push this using the CLI but am getting an error that says 'fatal: not a git repository (or any of the parent directories)'. I did try using the 'deploy to heroku' button on docker hub but this sends me to an error page. 

If you have any pointers that would be greatly appreciated, but also appreciate this is a personal project and it's not your job to spend time teaching newbies!

Cheers, 

Felix

EDIT:
scratch this I think I have figured it out! thank you

**Comment by stefansundin** [comment]
Great that you have figured it out. 👍 If you have any learnings to share then feel free to post it here for people who have similar problems.

I don't really recommend Heroku anymore since you have to pay for it. You can host it at fly.io for free.

**Reply by stefansundin** [reply]
I didn't realize that the Docker Hub page still had the "Deploy to Heroku" button. I updated it to just direct people to GitHub. Thanks for letting me know about this!
